### PR TITLE
refactor: use scheduler as dev portal

### PR DESCRIPTION
**Pull request by daskyrk**
## What type of PR is this?

- [ ] Feature
- [ ] Bugfix
- [ ] Test
- [ ] Documentation
- [x] Refactor
- [ ] Style
- [ ] Chore

## What this PR does / why we need it:
* Scheduler run server and provide proxy.
* Other modules run in watch mode and provide public files.
* Remove usage of .erda/config file

## Which issue(s) this PR fixes:

Fixes #

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [x] No

### CODE REVIEW

File: scheduler/package.json [hunk 1]
```diff
@@ -5,17 +5,20 @@
   "main": "src/index.js",
   "private": true,
   "scripts": {
-    "start": "nodemon -e js,json,env src/index.js --watch ../.env --watch src"
+    "start": "nodemon --delay 1 -e js,json,env src/index.js --watch ../public/static --watch src"
   },
   "author": "Erda-FE",
   "license": "AGPL",
   "dependencies": {
+    "dotenv": "^8.6.0",
```
Comment (McDaddy) [inline scheduler/package.json:13]: dotenv is not using in sheduler

File: scheduler/src/index.js [hunk 2]
```diff
@@ -12,44 +12,66 @@
...
+
+const prodModules = {};
+getDirectories(staticDir).forEach(m => {
+  prodModules[m] = true;
+});
+console.log('exist modules:', Object.keys(prodModules));
+
+
+const redirectMiddleware = async (ctx) => {
+  const [empty, scope, moduleName, ...rest] = ctx.path.split('/');
```
Comment (McDaddy) [inline scheduler/src/index.js:38]: if empty is an omit variable, better written as 
```
_empty  or
const [, scope, moduleName, ...rest] = ctx.path.split('/');
```

File: setup.js [hunk 3]
```diff
@@ -55,19 +54,17 @@ const setupShell = async (port) => {
 
 const initEnvFile = async () => {
   const envContent = `
-DEV_HOST=https://terminus-org.dev.terminus.io
-TEST_HOST=https://terminus-org.test.terminus.io
-SCHEDULER_HOST=http://localhost
+ERDA_DIR=${root}
+BACKEND_URL=https://terminus-org.dev.terminus.io
+SCHEDULER_HOST=https://dice.dev.terminus.io
```
Comment (McDaddy) [inline setup.js:59]: `_HOST` named config with protocol sounds a little tricky

File: scheduler/package.json (hunk 1 above)
Comment (daskyrk) [reply scheduler/package.json:13]: used now

**Review by McDaddy — APPROVED** [review]

**Pull request merged**